### ISSUE TITLE
Feature request: choose existing kernels when running code for the first time

### ISSUE THREAD

**Issue by dnowacki-usgs** [nteract/hydrogen#1100]
It would be great when running code in a file for the first time to let the user choose an existing running kernel in addition to the option to start a new one. My muscle memory presses cmd-alt-enter to run a code cell in a files when I often want to connect to an existing kernel. Having to cmd-shift-P and then search for "connect" is a small point of friction that would be nice to smooth out.

Basically, this is a request to add the contents of `Hydrogen: Connect To Existing Kernel` to `Hydrogen: Run`, `Hydrogen: Run Cell`, etc., which currently only show the contents of `Hydrogen: Start Local Kernel`.

**Comment by wadethestealth** [comment]
@BenRussert @aviatesk Whats your guys opinion on this?
